### DISCUSSION TITLE
Are there two filter setup equivalent between Nek5000 and NekRS?

### DISCUSSION THREAD

**Discussion by yukkiezy** [Nek5000/nekRS#584] [General]
I would like to find an equivalent filter setup in NekRS to match the filter I used to use in Nek5000
Are there two the same?
Nek5000
filtering = hpfrt
filterModes = 2
filterWeight = 5.0

NekRS
regularization = hpfrt + nModes=2 + scalingCoeff= 5.0

**Comment by stgeke** [comment]
Yes!
